tests/unit/test_attendance.py

```diff
@@ -0,0 +1,97 @@
+from tests import PMGTestCase
+from pmg.models import db, CommitteeMeeting, CommitteeMeetingAttendance, Committee, House, Province, Party, Member
+
+
+class TestCommitteeMeetingAttendance(PMGTestCase):
```

Comment:
Let's document:
This is testing:
- new data will show up on the page requesting "current" and NOT on "historical
- old data will show up on the page requesting "historical" and NOT on "current"